packages/i18n-calypso/src/i18n.js

```diff
@@ -168,13 +168,12 @@ I18N.prototype.emit = function ( ...args ) {
 
 /**
  * Formats numbers using locale settings and/or passed options.
- * @param   {string | number}  number to format (required)
- * @param   {number | Object}  options  Number of decimal places or options object (optional)
- * @param   {boolean}          forceLatin Whether to use latin numbers by default (optional. default = true)
- * @returns {string | number}  Formatted number as string, or original number if formatting fails. Null otherwise.
+ * @returns {string | number}  Formatted number as string, or original number if formatting fails
  */
-I18N.prototype.numberFormat = function ( number, options = {}, forceLatin = true ) {
-	const decimals = typeof options === 'number' ? options : options.decimals || 0;
+I18N.prototype.numberFormat = function (
+	number,
+	{ decimals = 0, forceLatin = true, numberFormatOptions = {} } = {}
```

Comment:
`decimals` defaulting to 0 is the identical behavior from previously, where decimals was defined as:

```
const decimals = typeof options === 'number' ? options : options.decimals || 0;
```

I am not sure if we should in fact set the default to 0. I'm guessing, for the most part, we don't need to i.e. how often do we want to render fractional numbers without the decimal points? If passed an integer, then `Intl.NumberFormat` will format it without decimals (so in that case it is also unnecessary).

My fear, with changing, is needing to inspect every instance of `numberFormat` call (with defaults) to confirm the numbers passed through aren't for whatever reason non-integer when expected whole.